source/plugins/go/src/oms.go

```diff
@@ -1603,6 +1603,9 @@ func PostDataHelper(tailPluginRecords []map[interface{}]interface{}) int {
 		var msgPackEntry MsgPackEntry
 
 		FlushedRecordsSize += float64(len(stringMap["LogEntry"]))
+		if ContainerLogSchemaV2 == true {
```

Comment:
1. Instead of ContainerLogSchemaV2 and we should check for KubernetesMetadataEnabled , rght?
2. Isnt that KubernetesMetadataEnabled  only for ContainerLogSchemaV2? if yes, then we can add check for KubernetesMetadataEnabled  otherwise we should add check for both  - 

if KubernetesMetadataEnabled  &&  ContainerLogSchemaV2 {
FlushedMetadataSize += float64(len(stringMap["KubernetesMetadata"]))
}

Reply:
changed to collect when KubernetesMetadataEnabled